### ISSUE TITLE
Public search vs. Operator search

### ISSUE THREAD

**Issue by acdoll** [ArctosDB/arctos#3105]
I'm getting different results on a search depending on whether or not I am logged in as an operator. I performed the following search to pull up the dunlin I collected in Barrow:
[https://arctos.database.museum/saved/Doll%5FDunlin%20collections](url)
(search terms: state/province: alaska, collector role: collector, collector: andrew charles doll)

Logged in as an operator, it returns the 60 bird catalog records and 1 parasite record in the DMNS collections. I wanted to share that search, but decided to test it first. In doing so as a public user, I only get 1 record for a parasitic jaeger I collected which went to UAM. I know why the jaeger doesn't appear in my search as an operator, but why don't the DMNS records appear in the public user search?
It seems to be an issue with including the collector in the search terms. I can publicly search for 'Calidris alpina' in 'Alaska' and find these specimens in the list. If I include myself a collector, I get nothing. If I try another collector, Charles D. Brower, I only get results from CHAS:Bird and not the 24 records he collected for DMNS.
@dustymc is there something restricting searches including Agents in the DMNS collections?

**Comment by dustymc** [comment]
Nope, just a typo - should be fixed and patched in, please confirm.

**Comment by acdoll** [comment]
That did it - thanks!
